Mathlib/Topology/Perfect.lean

```diff
@@ -52,7 +53,8 @@ accumulation point, perfect set, cantor-bendixson.
 
 open Topology Filter
 
-open TopologicalSpace Filter Set
+open Filter Set
+open TopologicalSpace (IsTopologicalBasis)
```

Comment:
Looks like `Filter` is open twice, all these lines can be replaced with just `open Topology Filter Set TopologicalSpace`